engines/osxnative/resultset.go

```diff
@@ -90,6 +91,12 @@ func (r resultset) ExtractFolder(path string, handler engines.FileHandler) error
 				return engines.ErrResourceNotFound
 			}
 
+			p, err = filepath.Rel(path, p)
+			if err != nil {
+				r.context.LogError(err)
+				return engines.ErrResourceNotFound
```

Comment:
Isn't this like an internal error of some kind? Ie. should we not crash the application? I'm not sure here...